resources/asciidoctor/lib/care_admonition/extension.rb

```diff
@@ -21,19 +48,83 @@ class CareAdmonition < Asciidoctor::Extensions::Group
   TEXT
+  BETA_SERVERLESS_TEXT = <<~TEXT.strip
+    This functionality is in beta in Elastic Cloud Serverless and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+  TEXT
+  BETA_ESS_TEXT = <<~TEXT.strip
+    This functionality is in beta in Elasticsearch Service and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+  TEXT
   DEV_DEFAULT_TEXT = <<~TEXT.strip
     This functionality is in development and may be changed or removed completely in a future release. These features are unsupported and not subject to the support SLA of official GA features.
   TEXT
+  DEV_SERVERLESS_TEXT = <<~TEXT.strip
+    This functionality is in development in Elastic Cloud Serverless and may be changed or removed completely in a future release. These features are unsupported and not subject to the support SLA of official GA features.
+  TEXT
+  DEV_ESS_TEXT = <<~TEXT.strip
+    This functionality is in development in Elasticsearch Service and may be changed or removed completely in a future release. These features are unsupported and not subject to the support SLA of official GA features.
+  TEXT
   PREVIEW_DEFAULT_TEXT = <<~TEXT.strip
     This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
   TEXT
+  PREVIEW_SERVERLESS_TEXT = <<~TEXT.strip
+    This functionality is in technical preview in Elastic Cloud Serverless and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
+  TEXT
+  PREVIEW_ESS_TEXT = <<~TEXT.strip
+    This functionality is in technical in preview in Elasticsearch Service and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
+  TEXT
+  DEPRECATED_ESS_TEXT = <<~TEXT.strip
+    This functionality is deprecated in Elasticsearch Service and will be removed in a future release.
+  TEXT
+  DEPRECATED_SERVERLESS_TEXT = <<~TEXT.strip
+    This functionality is deprecated in Elastic Cloud Serverless and will be removed in a future release.
+  TEXT
+  DISCONTINUED_ESS_TEXT = <<~TEXT.strip
+    This functionality is discontinued in Elasticsearch Service.
+  TEXT
+  DISCONTINUED_SERVERLESS_TEXT = <<~TEXT.strip
+    This functionality is discontinued in Elastic Cloud Serverless.
+  TEXT
```

Comment:
What is the difference between deprecated and discontinued? Can we make the discontinued description more specific?

Reply:
We haven't had "discontinued" before but it arose in discussions like https://github.com/elastic/kibana/issues/192292. Currently we just remove content when the pertinent feature goes from "deprecated" to discontinued/removed/no longer available. If our pages span multiple versions, however, or if we want to give folks a clear indication of when a feature will be removed, it seems like this might be something we'll want to have a way of calling out.